src/LendingPool.sol

```diff
@@ -511,17 +514,55 @@ contract LendingPool is LendingPoolGuardian, TrustedCreditor, DebtToken, Interes
      * Function will not revert, but transferAmount is always 0.
      * @dev Anyone (EOAs and contracts) can repay debt in the name of a Account.
      */
+
     function repay(uint256 amount, address account) external whenRepayNotPaused processInterests {
         uint256 accountDebt = maxWithdraw(account);
         uint256 transferAmount = accountDebt > amount ? amount : accountDebt;
 
+        _repay(transferAmount, transferAmount, account, msg.sender);
+    }
+
+    /**
+     * @notice Repays a portion of a user's debt in an auction.
+     * @dev This function allows a liquidator to repay a specified amount of debt for a user.
+     * @param amount The amount to be repaid, which can be at most the user's debt.
+     * @param account The address of the user whose debt is being repaid.
+     * @param bidder The address of the liquidator performing the repayment.
+     * @dev This function transfers tokens from the `bidder` to the contract, and then updates the user's account balance accordingly.
+     * @dev The `onlyLiquidator` modifier restricts this function to authorized liquidators.
```

Comment:
Remove this and next lines in natspec, no need to explain modifiers, events or add a second notice